src/engine/strat_engine/thinpool/thinpool.rs

```diff
@@ -417,9 +428,11 @@ impl ThinPool {
                        extend_size: DataBlocks,
                        backstore: &mut Backstore)
                        -> EngineResult<DataBlocks> {
+        let backstore_device = self.backstore_device;
```

Comment:
Forward-looking. Is correct within current context, but would seem unduly paranoid w/out expected changes.